app/controllers/api/v1/exams_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class ExamsController < ApplicationController
+      def index
+        @exams = Exam.all.map { |exam| { id: exam.id, difficulity: exam.difficulity, questions_count: exam.results.count } }
+        render json: @exams
+      end
+
+      def create
+        @exam = current_user.exams.create(exam_params)
+        questions = list_question(@exam)
+        values = questions.map { |question| { exam_id: @exam.id, question_id: question.id } }
+        Result.import values
+        render json: @exam.as_json.merge("questions_count" => questions.length)
+      end
+
+      def show
+        @a = Exam.find(params[:id])
+        @exam = @a.results.select(:*).joins(:question)
```

Comment:
mục đích select(:*) là gì ?

Reply:
Em cần show ra toàn bộ record của table Result nên em select(:*) đó anh

Reply:
vậy k dùng select(:*) là k lấy ra toàn bộ được hả :v